learners/demo_lessons.md

```diff
@@ -66,7 +66,7 @@ Use the links below to jump to the starting points information for a particular
-| [Data Analysis and Visualization in R](https://datacarpentry.org/R-ecology-lesson/index.html) |  \* [Introduction to R](https://datacarpentry.org/R-ecology-lesson/01-intro-to-r.html)<br/>\* [Starting with Data](https://datacarpentry.org/R-ecology-lesson/02-starting-with-data.html) | \* [Before We Start](https://www.datacarpentry.org/R-ecology-lesson/00-before-we-start.html) | [Manipulating Data](https://datacarpentry.org/R-ecology-lesson/03-dplyr.html) and [Data Visualization with ggplot2](https://datacarpentry.org/R-ecology-lesson/04-visualization-ggplot2.html) depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
+| [Data Analysis and Visualization in R](https://datacarpentry.org/R-ecology-lesson/index.html) |  \* [Data Visualization with ggplot2]https://datacarpentry.org/R-ecology-lesson-alternative/visualizing-ggplot.html) | \* [Introduction to R and RStudio](https://datacarpentry.org/R-ecology-lesson-alternative/introduction-r-rstudio.html) | The other episodes depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
 | [Data Management with SQL](https://www.datacarpentry.org/sql-ecology-lesson/index.html) | \* [Accessing Data With Queries](https://datacarpentry.org/sql-ecology-lesson/01-sql-basic-queries.html)<br/>\* [Aggregating and Grouping Data](https://datacarpentry.org/sql-ecology-lesson/02-sql-aggregation.html)<br/>\* [Combining Data With Joins](https://datacarpentry.org/sql-ecology-lesson/03-sql-joins.html) | \* [Databases using SQL](https://datacarpentry.github.io/sql-ecology-lesson/00-sql-introduction.html) |  |
 | [Data Analysis and Visualization in Python](https://www.datacarpentry.org/python-ecology-lesson/index.html) |  \* [Short Introduction to Programming](https://datacarpentry.org/python-ecology-lesson/01-short-introduction-to-Python.html)<br/> \* [Starting with Data](https://datacarpentry.org/python-ecology-lesson/02-starting-with-data.html)<br/> \* [Indexing, Slicing, and Subsetting](https://datacarpentry.org/python-ecology-lesson/03-index-slice-subset.html)<br/> \* [Data Types and Formats](https://datacarpentry.org/python-ecology-lesson/04-data-types-and-format.html)<br/> \* [Combining DataFrames with Pandas](https://datacarpentry.org/python-ecology-lesson/05-merging-data.html)<br/> \* [Data Workflows and Automation](https://datacarpentry.org/python-ecology-lesson/06-loops-and-functions.html) | \* [Before we Start](https://datacarpentry.org/python-ecology-lesson/00-before-we-start.html) | The other episodes depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
 
```

Comment:
It was recommended that we have a transition period and the following is my recommendation for implementing: 

```suggestion
::::::::::::::::::::::::::::::::::::: instructor

The following is no longer recommended for new trainees, but is eligible for trainees who have previously registered for it: 

|  Lesson   | Recommended Episodes | [**Not Eligible for Demos**](#ineligible) | Notes
|---------- |--------------- |--------------- |------------------------- |
| [Data Analysis and Visualization in R](https://datacarpentry.org/R-ecology-lesson/index.html) |  \* [Introduction to R](https://datacarpentry.org/R-ecology-lesson/01-intro-to-r.html)<br/>\* [Starting with Data](https://datacarpentry.org/R-ecology-lesson/02-starting-with-data.html) | \* [Before We Start](https://www.datacarpentry.org/R-ecology-lesson/00-before-we-start.html) | [Manipulating Data](https://datacarpentry.org/R-ecology-lesson/03-dplyr.html) and [Data Visualization with ggplot2](https://datacarpentry.org/R-ecology-lesson/04-visualization-ggplot2.html) depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |

:::::::::::::::::::::::::::::::::::::
```

 plus, an announcement to trainers to use the "instructor view" of this page for extra tips

Reply:
I like the suggestion, but the redesigned version has now replaced the previous content at https://datacarpentry.org/R-ecology-lesson/, which means that these links do not work anymore. I can set up another repository with the last version of the content before the updates were merged, and we could link there.

Reply:
I created a site using the last version before the redesign was merged: https://tobyhodges.github.io/R-ecology-lesson-previous/ You can use that URL to replace the DC site in your Instructor Note above, if you like, and I will be happy to commit the suggestion.

BTW if we are taking this route, we should open another issue to remind us to remove the note again in the future. Given the fairly prominent signposting of the transition over the last several months, I think this can be a relatively short transition period.

Reply:
I think I did the url replacement I think we should leave it for the people who have *already* [signed up](https://pad.carpentries.org/teaching-demos) before we merge this change? So we can make the new issue to change it on August 9?